Library/Homebrew/cmd/test-bot.rb

```diff
@@ -445,6 +445,10 @@ def formula formula_name
         test "brew", "test", "--verbose", formula_name if formula.test_defined?
         testable_dependents.each do |dependent|
           unless dependent.installed?
+            conflicts = dependent.conlicts.map { |c| Formulary.factory(c.name) }.select { |f| f.installed? }
```

Comment:
Typo `conlicts.map`. Did you test this locally?
Also, should put this before `install` and after `fetch`.